### PR TITLE
feat(api): add BOBA price feeds

### LINKED ISSUE #51: Add Boba feeds

### PR DESCRIPTION
**Pull request by Tommytrg**
Close #51

### CODE REVIEW

**Review by mariocao — COMMENTED** [review]
Is this PR still up to date?

File: packages/api/src/dataFeeds.json [hunk 1]
```diff
@@ -70,5 +70,29 @@
...
+    "abi": "./src/abi/EthUsd.json",
+    "address": "0x8936CC30c4EA38e269a8C3eCfC91D398a460d759",
+    "network": "omgx_rinkeby",
+    "name": "eth/usd",
+    "label": "$",
+    "pollingPeriod": 5000,
+    "witnetRequestBoard": {
+      "address": "0x9AEC8fDABa5AC2add0C6be7d833D05ed35240A3c",
+      "abi": "./src/abi/WitnetRequestBoardProxy.json"
+    }
```
Comment (aesedepece) [inline packages/api/src/dataFeeds.json:84]: Not needed anymore

File: packages/api/src/dataFeeds.json [hunk 2]
```diff
@@ -70,5 +70,29 @@
...
+    "abi": "./src/abi/BtcUsd.json",
+    "address": "0x3D4281b386f0dFD7fCeDC1E679d51bDc4B1306cD",
+    "network": "omgx_rinkeby",
+    "name": "btc/usd",
+    "label": "$",
+    "pollingPeriod": 5000,
+    "witnetRequestBoard": {
+      "address": "0x9AEC8fDABa5AC2add0C6be7d833D05ed35240A3c",
+      "abi": "./src/abi/WitnetRequestBoardProxy.json"
+    }
```
Comment (aesedepece) [inline packages/api/src/dataFeeds.json:96]: Same

File: packages/api/src/dataFeeds.json [hunk 3]
```diff
@@ -70,5 +70,29 @@
...
       "abi": "./src/abi/WitnetRequestBoardProxy.json"
     }
+  },
+  {
+    "abi": "./src/abi/EthUsd.json",
+    "address": "0x8936CC30c4EA38e269a8C3eCfC91D398a460d759",
+    "network": "omgx_rinkeby",
+    "name": "eth/usd",
+    "label": "$",
+    "pollingPeriod": 5000,
```
Comment (aesedepece) [inline packages/api/src/dataFeeds.json:80]: This period is just too low

**Review by aesedepece — CHANGES_REQUESTED** [review]

**Comment by Tommytrg** [comment]
> Is this PR still up to date?

Indeed, I've updated and rebased the PR

**Review by aesedepece — COMMENTED** [review]
Looks OK to me but let's wait for the final addresses!

File: packages/api/src/dataFeeds.json [hunk 4]
```diff
@@ -130,5 +130,38 @@
     "pollingPeriod": 15000,
     "color": "#be4141de",
     "blockExplorer": "https://testnet.confluxscan.io/address/{address}"
+  },
+  {
+    "feedFullName": "boba-rinkeby_omg-btc_6",
```
Comment (aesedepece) [inline packages/api/src/dataFeeds.json:135]:
```suggestion
    "feedFullName": "boba-rinkeby_omg-btc_9",
```

**Review by aesedepece — CHANGES_REQUESTED** [review]

**Comment by aesedepece** [comment]
Merged in af36b2fbb1826d03a4dadec6c7abb17cd58c540a

**Pull request merged**